### PR TITLE
cloud: pause or resume a cluster

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# Pause or Resume a TiDB Cluster
+
+TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.
+
+Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).
+
+> **Note:**
+>
+> Currently, you cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). Once a Developer Tier cluster remains idle for 24 hours, the cluster hibernates automatically, and you can resume it by [connecting to your cluster](/tidb-cloud/connect-to-tidb-cluster.md) or clicking **Resume** for the cluster in the TiDB Cloud console.
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:14]:
```suggestion
> **Note:**
>
> You cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). See [Automatic hibernation and resuming] for Developer Tier.
```

Comment (SunRunAway) [reply tidb-cloud/pause-or-resume-tidb-cluster.md:14]: How about putting this into Limitation section.

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Prerequisites
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:16]: Is it better to change it to "Limitations" and put it at the end of this doc?

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+- You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster.
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:18]:
```suggestion
<!--- - You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster. --->
```

Changefeed is not public.

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- You can pause your cluster only when it is in the **Normal** state. If a cluster is in other states such as **Importing** or **Scaling**, you must wait for the current operation to be completed before pausing the cluster.
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:19]:
```suggestion
- You can pause your cluster only when it is in the **Normal** state. If a cluster is in other states such as **Scaling**, you must wait for the current operation to be completed before pausing the cluster.
- You can not pause your cluster if the cluster is being imported, you can either wait for the import task finished or cancel the import task on TiDB Cloud UI.

```

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- You cannot pause your cluster when it is backing up data, you can either wait for the current backup jobs to be completed or [delete the running backup job](/tidb-cloud/backup-and-restore.md#delete-a-running-backup-job).
+
+## Pause a TiDB cluster
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:22]: Will we add Open API method if API becomes public?

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+When a cluster is paused, note the following:
+
+- TiDB Cloud stops collecting monitoring information of the cluster, you cannot read data from or write data to the cluster, and you cannot import or back up data.
+- TiDB Cloud only charges the cluster for storage cost and does not charge for any other costs.
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:27]:
```suggestion
- Only the following costs will be charged: 
    1. Node Storage Cost
    2. Data Backup Cost
    3.  Data Transfer – Load Balancing

```

File: tidb-cloud/pause-or-resume-tidb-cluster.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/pause-or-resume-tidb-cluster.md:16]: It depends on when we want users to notice this info, before the pausing, or after the pausing.

File: tidb-cloud/pause-or-resume-tidb-cluster.md (hunk 5 above)
Comment (qiancai) [reply tidb-cloud/pause-or-resume-tidb-cluster.md:23]: If we want users to try API,  we can add a link here. It depends on when API is ready for trial.

File: tidb-cloud/pause-or-resume-tidb-cluster.md (hunk 1 above)
Comment (qiancai) [reply tidb-cloud/pause-or-resume-tidb-cluster.md]: This is not a limitation for Dedicated Tiers, but only for Developer Tiers. What about still keeping it as a note and adding similar info to https://docs.pingcap.com/tidbcloud/select-cluster-tier#developer-tier-special-terms-and-conditions?

File: tidb-cloud/changefeed-overview.md [hunk 7]
```diff
@@ -10,3 +10,7 @@ TiDB Cloud provides the following changefeeds to help you stream data from TiDB
 - [Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md)
 
 To learn the billing for changefeeds in TiDB Cloud, see [Changefeed billing](/tidb-cloud/tidb-cloud-billing-tcu.md).
+
+> **Note:**
+>
+> You cannot [pause your cluster](/tidb-cloud/pause-or-resume-tidb-cluster.md) if it has any Changefeeds. You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster.
```
Comment (SunRunAway) [inline tidb-cloud/changefeed-overview.md:16]:
```suggestion
```

**Review by SunRunAway — APPROVED** [review]

File: tidb-cloud/changefeed-overview.md (hunk 7 above)
Comment (TomShawn) [inline tidb-cloud/changefeed-overview.md:16]:
```suggestion
> You cannot [pause your cluster](/tidb-cloud/pause-or-resume-tidb-cluster.md) if it has any changefeeds. You need to delete the existing changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster.
```

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.
+
+The pause does not affect your data stored in the cluster but only stops the monitoring information collection and computing resource consumption. After the pause, you can resume your cluster at any time.
+
+Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).
+
+> **Note:**
+>
+> You cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). See [Automatic hibernation and resuming](/tidb-cloud/select-cluster-tier.md#automatic-hibernation-and-resuming) for Developer Tier.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/pause-or-resume-tidb-cluster.md:16]: See https://github.com/pingcap/docs/pull/10031#discussion_r948635926. We will still keep the current sentences.

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by yuezhanglang — APPROVED** [review]
LGTM

**Review by Ivy-YinSu — APPROVED** [review]

File: tidb-cloud/select-cluster-tier.md [hunk 9]
```diff
@@ -81,6 +81,7 @@ Alternatively, you can log in to the TiDB Cloud console, and then click **Resume
 - You cannot create any changefeeds (Apache Kafka Sink and MySQL Sink) or use [TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview) to replicate incremental data.
 - You cannot use VPC Peering to connect to clusters.
 - You cannot scale clusters to larger storage, standard nodes, or increase the number of nodes.
+- You cannot [pause or resume a cluster](/tidb-cloud/pause-or-resume-tidb-cluster.md).
```
Comment (DanielZhangQD) [inline tidb-cloud/select-cluster-tier.md:84]: Why `cannot`?

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: tidb-cloud/select-cluster-tier.md (hunk 9 above)
Comment (SunRunAway) [inline tidb-cloud/select-cluster-tier.md:84]:
```suggestion
- You cannot [pause or resume a Developer Tier cluster](/tidb-cloud/pause-or-resume-tidb-cluster.md).
```

File: tidb-cloud/pause-or-resume-tidb-cluster.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+When a cluster is paused, note the following:
+
+- TiDB Cloud stops collecting monitoring information of the cluster.
+- You cannot read data from or write data to the cluster.
+- You cannot import or back up data.
+- Only the following costs will be charged:
+
+    - Node Storage Cost
+    - Data Backup Cost
+    - Data Transfer – Load Balancing
```
Comment (SunRunAway) [inline tidb-cloud/pause-or-resume-tidb-cluster.md:37]:
```suggestion
```
@DanielZhangQD and I have decided to remove this.

File: tidb-cloud/release-notes-2022.md [hunk 11]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## August 23, 2022
+
+**General changes**
+
+* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.2.0](https://docs.pingcap.com/tidb/stable/release-6.2.0).
```
Comment (Oreoxmt) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.2.0](https://docs.pingcap.com/tidb/dev/release-6.2.0).
```

Comment (qiancai) [reply tidb-cloud/release-notes-2022.md:15]: It should be https://docs.pingcap.com/tidb/v6.2/release-6.2.0

Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
* Upgrade the default TiDB version of new [Developer Tier](/tidb-cloud/select-cluster-tier.md#developer-tier) clusters from [v6.1.0](https://docs.pingcap.com/tidb/stable/release-6.1.0) to [v6.2.0](https://docs.pingcap.com/tidb/v6.2/release-6.2.0).
```

**Review by SunRunAway — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**